homeassistant/components/lamarzocco/number.py

```diff
@@ -121,3 +254,40 @@ async def async_set_native_value(self, value: float) -> None:
         """Set the value."""
         await self.entity_description.set_value_fn(self.coordinator, value)
         self.async_write_ha_state()
+
+
+class LaMarzoccoKeyNumberEntity(LaMarzoccoEntity, NumberEntity):
+    """Number representing espresso machine with key support."""
+
+    entity_description: LaMarzoccoKeyNumberEntityDescription
+
+    def __init__(
+        self,
+        coordinator: LaMarzoccoUpdateCoordinator,
+        description: LaMarzoccoKeyNumberEntityDescription,
+        key: int,
+    ) -> None:
+        """Initialize the entity."""
+        super().__init__(coordinator, description)
+        if key == 0:
```

Comment:
Please add a comment what the key stand for and maybe there is even a better name for it